src/integrate/totalvi/config.vsh.yaml

```diff
@@ -0,0 +1,125 @@
+functionality:
+  name: totalvi
+  namespace: "integrate"
+  version: "dev"
+  description: "Performs mapping to the reference by totalvi model: https://docs.scvi-tools.org/en/stable/tutorials/notebooks/scarches_scvi_tools.html#Reference-mapping-with-TOTALVI"
+  authors:
+    - __merge__: ../../authors/vladimir_shitov.yaml
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: "--query"
+          alternatives: ["-q"]
+          type: file
+          description: Input h5mu file
+          direction: input
+          required: true
+        - name: "--reference"
+          alternatives: ["-r"]
+          type: file
+          description: Input h5mu file
+          direction: input
+          required: true
+        - name: "--force_retrain"
+          alternatives: ["-f"]
+          type: boolean_true
+          description: If true, retrain the model and save it to reference_model_path
+        - name: "--query_modality"
+          type: string
+          default: "rna"
+          required: false
+        - name: "--query_proteins_key"
+          type: string
+          description: Name of the modality containing proteins in the query
+          required: false
+        - name: "--reference_modality"
+          type: string
+          default: "rna"
+          required: false
+        - name: "--reference_proteins_key"
+          type: string
+          description: Name of the modality containing proteins in the reference
+          required: false
+        - name: "--input_layer"
+          type: string
+          required: false
+          description: "Input layer to use. If None, X is used"
+        - name: "--obs_batch"
```

Comment:
I see that this argument is used in 

```python3
    scvi.model.TOTALVI.setup_anndata(
        adata_reference,
        batch_key=par["obs_batch"],
        protein_expression_obsm_key=par["reference_proteins_key"]
    )
```
Does this mean that this key needs to be present in the reference?